easybuild/easyconfigs/t/tensorboard/tensorboard-2.10.0-foss-2022a.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'tensorboard'
+version = '2.10.0'
+
+homepage = 'https://github.com/tensorflow/tensorboard'
+description = """TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs
+and graphs."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('protobuf-python', '3.19.4'),
+]
+
+exts_list = [
+    ('absl-py', '1.2.0', {
+        'modulename': 'absl',
+        'checksums': ['f568809938c49abbda89826223c992b630afd23c638160ad7840cfe347710d97'],
+    }),
+    ('cachetools', '5.2.0', {
+        'checksums': ['6a94c6402995a99c3970cc7e4884bb60b4a8639938157eeed436098bf9831757'],
+    }),
+    ('pyasn1-modules', '0.2.8', {
+        'checksums': ['905f84c712230b2c592c19470d3ca8d552de726050d1d1716282a1f6146be65e'],
+    }),
+    ('rsa', '4.9', {
+        'checksums': ['e38464a49c6c85d7f1351b0126661487a7e0a14a50f1675ec50eb34d4f20ef21'],
+    }),
+    ('google-auth', '2.11.1', {
+        'modulename': 'google.auth',
+        'checksums': ['516e6623038b81430dd062a1a25ecd24f173d7c15cdf4e48a9e78bc87e97aeec'],
+    }),
+    ('oauthlib', '3.2.1', {
+        'checksums': ['1565237372795bf6ee3e5aba5e2a85bd5a65d0e2aa5c628b9a97b7d7a0da3721'],
+    }),
+    ('requests-oauthlib', '1.3.1', {
+        'checksums': ['75beac4a47881eeb94d5ea5d6ad31ef88856affe2332b9aafb52c6452ccf0d7a'],
+    }),
+    ('google-auth-oauthlib', '0.4.6', {
+        'checksums': ['a90a072f6993f2c327067bf65270046384cda5a8ecb20b94ea9a687f1f233a7a'],
+    }),
+    ('grpcio', '1.49.1', {
+        'modulename': 'grpc',
+        'preinstallopts': "export GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS=%(parallel)s && ",
+        'checksums': ['d4725fc9ec8e8822906ae26bb26f5546891aa7fbc3443de970cc556d43a5c99f'],
+    }),
+    ('importlib_metadata', '4.12.0', {
```

Comment:
@surak 
`importlib_metadata` (version 4.11.3) is already included in Python 3.10.4 
is that not recent enough?

Reply:
Yes, it is. It seems that eb can't work with --update-pr if you used --read-only-installdir before, but you are correct and it works just fine